lib/mappings/mappings.js

```diff
@@ -12,10 +12,9 @@ const amendMappingsBasedOnNyplSource = (data, nyplSource) => {
     const revised = Object.assign({}, data[name])
     if (revised.paths) {
       // Remove unmatching paths:
-      revised.paths = revised.paths.reduce((a, p) => {
-        if (!p.nyplSources || (Array.isArray(p.nyplSources) && sourceMatches(nyplSource, p.nyplSources))) a.push(p)
```

Comment:
a classic reduce to an array/filter switcheroo. definitely did at least one of those on my last ticket.